src/twisted/trial/_asynctest.py

```diff
@@ -58,6 +58,14 @@ def __init__(self, methodName="runTest"):
         """
         super().__init__(methodName)
 
+        # Acquire and store real reactor so that it does not interfere with any
+        # patching to the reactor done by the user.
+        from twisted.internet import reactor
+
+        # A unique name must be used in order not to clash with user code that
+        # sets their own attributes.
+        self._twistedPrivateReactor = reactor
```

Comment:
As this instance is only used to schedule tasks and not do do any I/O operations, maybe this can be called a "scheduler"

```suggestion
        self._twistedPrivateScheduler = reactor
```

Reply:
Fixed, thanks.